tests/rptest/tests/debug_bundle_test.py

```diff
@@ -0,0 +1,189 @@
+            auth_tuple = (super_username, super_password)
+
+        self.redpanda.set_security_settings(security)
+        self.redpanda.start()
+        self._create_initial_topics()
+
+        msg_count = 1000
+        producer = RpkProducer(self.test_context,
+                               self.redpanda,
+                               self.topic,
+                               msg_size=12800,
+                               msg_count=msg_count,
+                               acks=-1,
+                               auth_tuple=self.redpanda.SUPERUSER_CREDENTIALS
+                               if use_basic_auth else None)
+        producer.start()
+        consumer = RpkConsumer(self.test_context,
+                               self.redpanda,
+                               self.topic,
+                               group="debug-bundle-group",
+                               num_msgs=msg_count,
+                               auth_tuple=self.redpanda.SUPERUSER_CREDENTIALS
+                               if use_basic_auth else None)
+        consumer.start()
+
+        wait_until(lambda: consumer.message_count > msg_count / 2,
+                   timeout_sec=30,
+                   backoff_sec=1)
+
+        node = self.redpanda.started_nodes()[0]
+        default_bundle_name = "debug-bundle.zip"
+
+        admin = Admin(self.redpanda)
+        journalctl_date = datetime.now().strftime("%Y-%m-%d")
+        res = admin.start_debug_bundle(node=node,
+                                       logs_since=journalctl_date,
+                                       logs_size_limit="100MiB",
+                                       metrics_interval="10s",
+                                       auth=auth_tuple)
+        assert res.status_code == requests.codes.accepted
+
+        try:
+            admin.start_debug_bundle(node=node)
+        except HTTPError as ex:
+            # Expect HTTP Error 429 since a bundle is already running
+            if ex.response.status_code == requests.codes.too_many_requests:
+                pass
+            else:
+                raise
+
+        def bundle_status():
+            try:
+                res = admin.debug_bundle_status(filename=default_bundle_name,
+                                                node=node,
+                                                auth=auth_tuple)
+                # OK status means the bundle is available for download
+                return res.status_code == requests.codes.ok
+            except HTTPError as ex:
+                # Not found status means the bundle is in progress but not yet ready
+                if ex.response.status_code == requests.codes.not_found:
+                    return False
+                # Gone status means the bundle is not in progress or it is not on disk
+                elif ex.response.status_code == requests.codes.gone:
+                    return False
+                else:
+                    raise
```

Comment:
In the scenario being tested on line 130, a response of `410 - GONE` would actually be an error.